notebooks/tel_and_site/subsys_req_ver/m1m3/SITCOM-756_analysis.ipynb

```diff
@@ -0,0 +1,701 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #121.            ax.set_ylabel("Frequency")
</pre><p>If I understand correctly what you call Frequency here is in fact "Counts" in histogram bins. Frequency may be misleading. </p><p>Perhaps change to "counts / {bin width}" </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-sitcom/notebooks_vandv/pull/135/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>